plugin/trino-delta-lake/pom.xml

```diff
@@ -26,6 +26,18 @@
         <air.test.parallel>instances</air.test.parallel>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.google.api-client</groupId>
+                <artifactId>google-api-client-bom</artifactId>
+                <type>pom</type>
+                <version>1.30.1</version>
```

Comment:
sheets connector also uses google-api-client but with a different version.

i think that `>google-api-client-bom` import should go into root pom and replace the sheets' version